app/dashboard/posts/create/action/store.ts

```diff
@@ -14,7 +14,10 @@ import { form } from '@/utils/form';
 
 import { multiStepFormStore } from './multiStepFormStore';
 
-let createdPlaceResult: Awaited<ReturnType<typeof place.create>>;
+type CreatedPlaceResultResponse = Awaited<ReturnType<typeof place.create>>;
+
+let createdPlaceResult: CreatedPlaceResultResponse =
+  {} as CreatedPlaceResultResponse;
```

Comment:
I instantiated this variable because it was showing a Typescript error on `/dashboard/posts/create.page.tsx` when trying to access `createdPlaceResult` properties